Products/ZenUI3/browser/resources/js/zenoss/form/configPropertyPanel.js

```diff
@@ -298,8 +298,8 @@
             Ext.applyIf(config, {
                 stateId: config.id || 'config_property_grid',
                 sm: Ext.create('Zenoss.SingleRowSelectionModel', {}),
-                tbar: {
-                    height: 45,
+                dockedItems:[{
```

Comment:
Using "dockedItems" instead of "tbar" to use "largetoolbar" component and make layout works right.